R/retry.R

```diff
@@ -0,0 +1,58 @@
+#' Retry a request until it succeeds.
+#'
+#' Safely retry a request until it succeeds (returns an HTTP status code
+#' below 400). It is designed to be kind to the server: after each failure
+#' randomly waits up to twice as long. (Technically it uses exponential
+#' backoff with jitter, using the approach outlined in
+#' \url{https://www.awsarchitectureblog.com/2015/03/backoff.html.})
+#'
+#' @inheritParams VERB
+#' @inheritParams GET
+#' @inheritParams POST
+#' @param times Maximum number of requests to attempt.
+#' @param pause_base,pause_cap This method uses exponential back-off of with
```

Comment:
two nits: 
 * spurious `of`
 * space after `pause_base`? (not actually sure if you want one)